.github/workflows/dotcom-acceptance-tests-all.yml

```diff
@@ -0,0 +1,58 @@
+name: Dotcom Acceptance Tests (All)
+
+on:
+  schedule:
+    #         ┌───────────── minute (0 - 59)
+    #         │ ┌───────────── hour (0 - 23)
+    #         │ │ ┌───────────── day of the month (1 - 31)
+    #         │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #         │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #         * * * * *
+    - cron:  '*/15 * * * *'
+
+jobs:
+
+  acceptance-tests-anonymous:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 2
+      - name: Acceptance Tests (Anonymous)
+        uses: terraformtesting/acceptance-tests@v2.0.0
+        with:
+          TF_LOG: DEBUG
```

Comment:
Am I right that the logs here will then end up somewhere on GitHub Action's "terminal output"? I'm only asking because the log _may_ contain some sensitive data. I know that GHA will mask known secrets automatically, but I just wanted to point this out in case it has any other implications.

Also I'm not sure how useful it really is to have the log mixed with the test output, instead of logging into a file. I guess the downside of logging into a file though is that GHA won't mask the secrets and the only way of exposing that file would be through artifacts?

Reply:
Thanks for catching this.  I found this output helpful when debugging while writing tests / fixing bugs.  This should be `INFO` by default and can be toggled while iterating on PRs.